tdrs-backend/tdpservice/parsers/test/test_models.py

```diff
@@ -21,3 +21,8 @@ def test_parser_error_rpt_month_name(parser_error_instance):
     """Test that the parser error instance is created."""
     parser_error_instance.rpt_month_year = 202001
     assert parser_error_instance.rpt_month_name == "January"
+
+
+@pytest.mark.django_db
+def test_datafile_summary_status_approved():
+    pass
```

Comment:
This was the test I meant. Vestigial or did you want to flesh this out?

Reply:
vestigial, before i realize the case i needed _was_ covered, my code just wasn't triggering the failure properly 🤦 

good catch, thank you